Tests/Functional/ViewHelpers/Variable/MapViewHelperTest.php

```diff
@@ -4,134 +4,147 @@
 
 namespace SMS\FluidComponents\Tests\Functional\ViewHelpers\Variable;
 
+use Generator;
 use PHPUnit\Framework\Attributes\DataProvider;
 use PHPUnit\Framework\Attributes\Test;
+use stdClass;
+use TYPO3\CMS\Core\Information\Typo3Version;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+use TYPO3\CMS\Core\View\FluidViewAdapter;
+use TYPO3\CMS\Core\View\ViewFactoryData;
+use TYPO3\CMS\Core\View\ViewFactoryInterface;
 use TYPO3\CMS\Fluid\View\TemplateView;
 use TYPO3\TestingFramework\Core\Functional\FunctionalTestCase;
 
 class MapViewHelperTest extends FunctionalTestCase
 {
     protected bool $initializeDatabase = false;
     protected array $testExtensionsToLoad = [
-        'typo3conf/ext/fluid_components'
+        'typo3conf/ext/fluid_components',
     ];
 
-    public static function renderDataProvider(): \Generator
+    public static function renderDataProvider(): Generator
     {
         $input = [
             0 => [
                 'sourceSimpleField' => 'sourceSimpleValue',
                 'sourcePathField' => [
-                    'path' => 'sourcePathValue'
+                    'path' => 'sourcePathValue',
                 ],
                 'keepField1' => 'keepValue1',
                 'keepField2' => 'keepValue2',
-            ]
+            ],
         ];
 
-        $obj = new \stdClass;
+        $obj = new stdClass;
         $obj->sourceSimpleField = 'sourceSimpleValue';
-        $obj->sourcePathField = new \stdClass;
+        $obj->sourcePathField = new stdClass;
         $obj->sourcePathField->path = 'sourcePathValue';
         $obj->keepField1 = 'keepValue1';
         $obj->keepField2 = 'keepValue2';
 
         yield 'array of objects' => [
             [0 => $obj],
-            '{dataSource -> fc:variable.map(fieldMapping: {targetSimpleField: "sourceSimpleField", targetPathField: "sourcePathField.path"}, keepFields: "keepField1, keepField2") -> f:variable(name: "dataTarget")}',
+            'workaround {dataSource -> fc:variable.map(fieldMapping: {targetSimpleField: "sourceSimpleField", targetPathField: "sourcePathField.path"}, keepFields: "keepField1, keepField2") -> f:variable(name: "dataTarget")}',
```

Comment:
I don't get it. :)

Reply:
Oh that was a strange behavior - i had to add any character to the template in order to get the test passed and forgot to remove this after fixing the underlying issue.